driver/test/src/main/java/org/opennms/alec/driver/test/TestDriver.java

```diff
@@ -265,13 +286,13 @@ private void printTick(long tick, long firstTimestamp, long lastTimeStamp, long
             return;
         }
         double percentageComplete = ((tick - firstTimestamp) / (double)(lastTimeStamp - firstTimestamp)) * 100d;
-        System.out.printf("Tick at %s (%d) - %.2f%% complete - %s elapsed\n", new Date(tick), tick,
```

Comment:
The usage of `System.out` is intentional in this class, since it's used in Karaf shell commands.